addon/components/bs-form/element/layout/horizontal/checkbox.hbs

```diff
@@ -8,11 +8,11 @@
     {{component @errorsComponent}}
     {{component @helpTextComponent}}
   {{else}}
-    <div class="form-check">
+    <div class="{{if this.isSwitch "custom-control custom-switch" "form-check"}}">
```

Comment:
I would suggest to express the same without introducing a CP in the backing class, by using the `bs-eq` helper here. Would ease the eventual transition to a template-only component later. Same for the other components.

In general we started to move more of the very basic derived state like this into the template, to reduce the need to add trivial CPs/getters to the class, again a.o. with the goal to use template-only components wherever possible.